kotlin/services/elasticbeanstalk/Readme.md

```diff
@@ -0,0 +1,62 @@
+# AWS Elastic Beanstalk Kotlin code examples
+
+## Overview
+This README discusses how to run and test the Kotlin code examples for AWS Elastic Beanstalk.
+
+AWS Elastic Beanstalk is an easy-to-use service for deploying and scaling web applications.
+
+## ⚠️ Important
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+*  We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single action
+
+The following examples use the **ElasticBeanstalkClient** object:
+
+- [Creating an AWS Elastic Beanstalk application](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/elasticbeanstalk/src/main/kotlin/com/aws/example/CreateApplication.kt) (CreateApplication command)
+- [Creating an AWS Elastic Beanstalk environment](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/elasticbeanstalk/src/main/kotlin/com/aws/example/CreateEnvironment.kt) (CreateEnvironment command)
+- [Deleting an AWS Elastic Beanstalk application](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/glue/src/main/kotlin/com/aws/example/DeleteApplication.kt) (DeleteApplication command)
+- [Describing an AWS Elastic Beanstalk application](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/glue/src/main/kotlin/com/aws/example/DescribeApplication.kt) (DescribeApplication command)
+- [Describing AWS Elastic Beanstalk configuration options](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/glue/src/main/kotlin/com/aws/example/DescribeConfigurationOptions.kt) (DescribeConfigurationOptions command)
+- [Describing an AWS Elastic Beanstalk environment](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/kotlin/services/glue/src/main/kotlin/com/aws/example/DescribeEnvironment.kt) (DescribeEnvironment command)
+
+
+## Running the AWS Elastic Beanstalk Kotlin files
+
+**IMPORTANT**
+
+The Kotlin examples perform AWS operations for the account and AWS Region for which you've specified credentials, and you may incur AWS service charges by running them. See the [AWS Pricing page](https://aws.amazon.com/pricing/) for details about the charges you can expect for a given service and operation.
```

Comment:
The SDK for Kotlin examples perform AWS operations for the account and AWS Region for which you've specified credentials. Running these examples might incur charges on your account. For details about the charges you can expect for a given service and API operation, see the [AWS Pricing page](https://aws.amazon.com/pricing/).

Reply:
fixed